Description: Updates POT file creation date to more recent one
Author: Aaser Abd-el Sabour <proman4713@gmail.com>
Last-Update: 2026-09-06 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: python-apt-abstract_any/po/python-apt.pot
===================================================================
--- python-apt-abstract_any.orig/po/python-apt.pot
+++ python-apt-abstract_any/po/python-apt.pot
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2025-01-28 18:47+0100\n"
+"POT-Creation-Date: 2026-08-26 22:20+0300\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
